src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -594,11 +594,11 @@ class MavlinkStreamSysStatus : public MavlinkStream
 			bat_msg.current_battery = (battery_status.connected) ? battery_status.current_filtered_a * 100 : -1;
 			bat_msg.battery_remaining = (battery_status.connected) ? ceilf(battery_status.remaining * 100.0f) : -1;
 			bat_msg.temperature = (battery_status.connected) ? (int16_t)battery_status.temperature : INT16_MAX;
+			bat_msg.time_remaining = (battery_status.connected) ? ((int32_t)battery_status.run_time_to_empty * 60) : 0;
```

Comment:
This isn't working for some reason. If I do this 
`bat_msg.energy_consumed = (battery_status.connected) ? ((int32_t)battery_status.run_time_to_empty * 60) : 0;`
Then energy_consumed, another mavlink int32 comes through correct. But time_remaining with the same code does not work. At least in today's build of QGC. 

![image](https://user-images.githubusercontent.com/2019539/40028515-75b135ca-579c-11e8-9935-ac0286ee69a2.png)

Reply:
From a log (or listener) is battery_status.run_time_to_empty otherwise correct?

Reply:
Yes, it is the correct value as reported by the fuel gauge.